api/v1alpha4/metal3machine_types.go

```diff
@@ -62,6 +62,13 @@ type Metal3MachineSpec struct {
 	// NetworkData is an object storing the reference to the secret containing the
 	// network data given by the user.
 	NetworkData *corev1.SecretReference `json:"networkData,omitempty"`
+
+	// When set to disabled, automated cleaning will be skipped
+	// during provisioning and deprovisioning.
```

Comment:
Maybe it would be good to add comment about what does default "metadata" value stands for? I understand that if `AutomatedCleaningMode` is set to "metadata" disk will be cleaned, but why not use disabled/enabled?

Reply:
Originally it was decided to be like this in BMO repo and the reason was in the future we might want to select particular disk from several to be deleted. For that reason, simply using disabled wasn't enough, because we can't specify which disk we would like to wipe out. Also, that was done to align with Ironic, because Ironic also has metadata option, and we might want to use that in the future.